lib/travis/build/script/r.rb

```diff
@@ -39,7 +44,7 @@ def export
           sh.export 'TRAVIS_R_VERSION', 'release', echo: false
         end
 
-        def setup
+        def configure
```

Comment:
will this break folks who are currently overriding `setup` in a `language: r` build, or have their own `configure` step?